internal/shim-kvm/src/hostcall.rs

```diff
@@ -519,17 +519,17 @@ impl Platform for UserMemScope {
     /// * not borrowed already
     /// and registers the memory as borrowed mutably.
     ///
-    /// Returns a mutable borrow if valid, otherwise [`EINVAL`](libc::EINVAL).
+    /// Returns a mutable borrow if valid, otherwise [`EINVAL`](https://man7.org/linux/man-pages/man3/errno.3.html).
     ///
     /// [the ptr module documentation]: core::ptr#safety
     #[inline]
     fn validate_mut<T>(&self, ptr: usize) -> Result<&mut T, c_int> {
-        is_aligned_non_null::<T>(ptr).ok_or(EINVAL)?;
+        is_aligned_non_null::<T>(ptr).ok_or(libc::EINVAL)?;
```

Comment:
Why this change? As a general rule, I think we should bring these kind of constants (e.g. errno or flags) into scope for clarity.

Reply:
I decided whether or not to bring things into scope based on how many items were being used and how many times each were being used. I don't mind bringing them all into scope if we have a blanket policy for that, though sometimes it might result in very long `use` statements.